week-2/implement/vat.js

```diff
@@ -8,3 +8,8 @@
 // Given a number,
 // When I call this function with a number
 // Then it returns the new price with VAT added on
+function vat(p,v){
+
+    return p*v ;
+}
+console.log(vat(50,1.2));
```

Comment:
BRILLIANT, however this will give you the total price, including VAT, (but not the original price plus the VAT.)